frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```
@@ -33,53 +33,54 @@
+50.  [console.project-overview/utilization-item](#consoleproject-overviewutilization-item)
+51.  [console.pvc/alert](#consolepvcalert)
+52.  [console.pvc/create-prop](#consolepvccreate-prop)
+53.  [console.pvc/delete](#consolepvcdelete)
+54.  [console.pvc/status](#consolepvcstatus)
+55.  [console.redux-reducer](#consoleredux-reducer)
+56.  [console.resource/create](#consoleresourcecreate)
+57.  [console.resource/details-item](#consoleresourcedetails-item)
+58.  [console.storage-class/provisioner](#consolestorage-classprovisioner)
+59.  [console.storage-provider](#consolestorage-provider)
+60.  [console.tab](#consoletab)
+61.  [console.tab/horizontalNav](#consoletabhorizontalNav)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid
Expected: `#consoletabhorizontalnav`; Actual: `#consoletabhorizontalNav`

(MD051, link-fragments)